Mobile: fixed overlapping text in dropbox sync page

ishantgupta777 opened the pull request:
closes #2843 

I fixed the issue of text overlapping and added the case of the dropbox sync page in note list component. 
@PackElend  please label me.

laurent22 commented:
Thanks, but I'm wondering what was causing this bug, did you figure it out? I'd rather avoid having Dropbox specific actions just to go around this bug, so I'm wondering if there's a better fix.

ishantgupta777 commented:
> Thanks, but I'm wondering what was causing this bug, did you figure it out? I'd rather avoid having Dropbox specific actions just to go around this bug, so I'm wondering if there's a better fix.

Sir this but was causing because, when there is no notes or notebook, then there is a message saying ( you have 0 notes/ notebook, clock to add ) ,  which is also coming in Dropbox page,  because every other sync settings doesn't require it's extra page, only Dropbox require, that's why it's only problem with Dropbox. 
I checked but couldn't find any state or method to find which page I'm on. That's why I created Dropbox page global statw. If there is any other way, please tell, I'll fix it.

ishantgupta777 commented:
> Thanks, but I'm wondering what was causing this bug, did you figure it out? I'd rather avoid having Dropbox specific actions just to go around this bug, so I'm wondering if there's a better fix.

Sir, you were right, there was no need for new action, route name is already present.  I fixed it and updated the new code, please have a look.

laurent22 commented:
What I don't get is what is the note list doing in the Dropbox login screen? Is it possible to make sure it's just not there (instead of modifying note list)?

ishantgupta777 commented:
> What I don't get is what is the note list doing in the Dropbox login screen? Is it possible to make sure it's just not there (instead of modifying note list)?

Sir, I finally found the main issue, The thing was, there was no background color to the container to the dropbox screen, so actually note-list wasn't there, it's actually below that screen, that's why it's visible. So I fixed it by adding a background color.

laurent22 commented:
> The thing was, there was no background color to the container to the dropbox screen, so actually note-list wasn't there, it's actually below that screen, that's why it's visible. So I fixed it by adding a background color.

Ah yes, that makes perfect sense. The note list is always present for optimisation purposes so indeed if a screen doesn't have a background, it would show below it. Thanks for the fix!
